jax/experimental/array.py

```diff
@@ -13,6 +13,7 @@
 # limitations under the License.
 
 from __future__ import annotations
+from email.mime import base
```

Comment:
What's `email.mime`?

Reply:
Gah, I have to set up vscode to not try to automatically add imports...

I suspect I typed `base` and hit tab, so it helpfully found something named `base` to import for me